packages/mermaid/src/schemas/config.schema.yaml

```
@@ -876,6 +876,13 @@ $defs: # JSON Schema definition (maybe we should move these to a separate file)
         description: |
           Axial position of slice's label from zero at the center to 1 at the outside edges.
         default: 0.75
+      percentageDecimals:
+        type: number
```

This is very very minor, but we can make this an `integer`, just to make the generated documentation slightly better!

```suggestion
        type: integer
```